hail/www/index.md

```diff
@@ -15,7 +15,7 @@ Hail is actively developed with new features and performance improvements integr
 
 ### Community
 
-Hail has been widely adopted in academia and industry, including as the analysis platform for the [genome aggregation database](https://gnomad.broadinstitute.org) and [UK Biobank rapid GWAS](https://www.nealelab.is/uk-biobank). Learn more about [Hail-powered science](references.md).
+Hail has been widely adopted in academia and industry, including as the analysis platform for the [genome aggregation database](https://gnomad.broadinstitute.org) and [UK Biobank rapid GWAS](https://www.nealelab.is/uk-biobank). Learn more about [Hail-powered science](https://hail.is/references.html).
```

Comment:
we should probably prefer `references.html` so that it works on, e.g. the locally built docs too